### PR TITLE
K8s: 7.2.4-2 Quartz release docs

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Staged preview: 

- https://docs.redis.com/staging/release-k8s-quartz/kubernetes/release-notes/7-2-4-releases/7-2-4-2/
- https://docs.redis.com/staging/release-k8s-quartz/kubernetes/re-clusters/auto-tiering/
- https://docs.redis.com/staging/release-k8s-quartz/kubernetes/recommendations/
- https://docs.redis.com/staging/release-k8s-quartz/kubernetes/reference/supported_k8s_distributions/

### CODE REVIEW

File: content/kubernetes/reference/supported_k8s_distributions.md [hunk 1]
```diff
@@ -44,94 +44,100 @@ Any distribution not listed below is not supported for production workloads.
 For details on this platform, see the Kubernetes [documentation](https://kubernetes.io/docs/home/supported-doc-versions/).
 
 {{<table-scrollable>}}
-|  | <nobr>**6.4.2-6**</nobr> | <nobr>**6.4.2-5**</nobr> | <nobr>**6.2.18-41**</nobr> | <nobr>**6.2.18-3**</nobr> | <nobr>**6.2.12-1**</nobr> | <nobr>**6.2.10-45**</nobr> | <nobr>**6.2.10-34**</nobr> | <nobr>**6.2.10-4**</nobr> | <nobr>**6.2.8-15**</nobr> | <nobr>**6.2.8-2**</nobr> | <nobr>**6.2.4-1**</nobr> |
```
Comment (zcahana) [inline content/kubernetes/reference/supported_k8s_distributions.md:47]: +1 for the new visual style for this page.
Did you also consider to inverse the versions order, both horizontal and vertical? i.e., have the operator versions increase left to right, and the k8s versions increase top to bottom? To my eyes this seems a bit more natural, not sure though. From a quick glance at other "compatibility matrices" this also seems like a more common way to lay it.

File: content/kubernetes/reference/supported_k8s_distributions.md [hunk 2]
```diff
@@ -140,50 +146,71 @@ For details on this platform, see the [GKE documentation](https://cloud.google.c
...
+
+{{<table-scrollable>}}
+|  | **<nobr>**7.2.4**</nobr>** | **<nobr>**6.4.2-6**</nobr>** | **<nobr>**6.4.2-5**</nobr>** | **<nobr>**6.2.18-41**</nobr>** |
+|---|---|---|---|---|
+| **1.24** | <span title="Supported">&#x2705;</span> |  |  |  |
+|**1.23** | <span title="Supported">&#x2705;</span> |  |  |  |
+| **1.22** | <span title="Supported">&#x2705;</span> |  |  |  |
+{{</table-scrollable>}}
+
+#### TKG 1.21
```
Comment (zcahana) [inline content/kubernetes/reference/supported_k8s_distributions.md:181]: I believe TKG 1.21/1.22 support hasn't been completed yet, and I'm not sure it'll complete before Quartz goes public. AFAIK the intent is to release without support statements for 1.21/1.22 and update that when support is established.

Also, TKG 1.21 only support k8s 1.24 (and not 1.22/1.23) and we'll only certify it on k8s 1.24 accordingly.
https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/2.1/tkg-deploy-mc-21/mgmt-release-notes.html#supported-kubernetes-versions-in-tanzu-kubernetes-grid-v21-5

File: content/kubernetes/reference/supported_k8s_distributions.md (hunk 1 above)
Comment (kaitlynmichael) [reply content/kubernetes/reference/supported_k8s_distributions.md:47]: We wanted the most recent to be the first thing the customer sees. Also if we did left to right, customers would have to scroll to see the most recent version.

**Review by cmilesb — APPROVED** [review]

File: content/kubernetes/reference/supported_k8s_distributions.md (hunk 2 above)
Comment (zcahana) [reply content/kubernetes/reference/supported_k8s_distributions.md]: As far as I know yes, but @yuvallevy2 can you keep me honest on this?

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+---
+
+## Highlights
+
+This version offers:
+
+- Support for [Redis Enterprise 7.2.4]({{<relref "/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52">}})
+
+- New default engine, Speedb, for Auto Tiering (formerly Redis on Flash)
+
```
Comment (bascar) [inline content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md:21]: Change "This version offers" to 
This release had the primary purpose of supporting the update of RS 7.2, which is a major Redis Enterprise Software update. As such, we reduced the scope of changes to support the underlying change of RS 7.2 and a few significant fixes. The following are the notable changes, except the regular update to the changing Kubernetes versions landscape.

**Review by bascar — DISMISSED** [review]
Change the highlights section, otherwise LGTM! Thank you, I know this was a big release.

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md (hunk 3 above)
Comment (kaitlynmichael) [reply content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md]: Changes approved by Brad via Slack

**Pull request merged**